doc/dev/developer_guide/index.rst

```diff
@@ -791,6 +791,59 @@ the `cram task`_.
 .. _`cram`: https://bitheap.org/cram/
 .. _`cram task`: https://github.com/ceph/ceph/blob/master/qa/tasks/cram.py
 
+Unit testing of python modules
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Most python modules can be found under ``./src/pybind/``.
+
+If they have any python tests you can run them with the help of **tox**.
+**tox** itself is a generic virtualenv management and test command line tool
```

Comment:
i am not sure if this is accurate. `ceph_argparse.py`, `ceph_volume_client.py`  and `ceph_daemon.py` are also python modules actively used by ceph. in which, `ceph_argparse.py` and `ceph_volume_client.py` are using nosetests. while `ceph_volume_client.py` is using `qa/tasks/cephfs/test_volume_client.py` for its testing.

strictly speaking we can use tox for testing them as well. but i don't think this is the intention of this section. probably we probably we can limit the scope of "python modules", like "Tox based testing of python modules". 

and change

> If they have any python tests you can run them with the help of **tox**.

to

> Quite a few of them utilize **tox** for performing unit tests.